eyeshade/workers/reports.js

```diff
@@ -855,7 +855,7 @@ const referralStatement = async (debug, runtime, owner, summaryP) => {
   ])
   referralTotals.forEach((total) => {
     total.publisher = total._id
-    total.probi = new BigNumber(total.probi)
+    total.probi = new BigNumber(total.probi.toString())
```

Comment:
i think you'll need the same fix at line 890:

        summary.probi = new BigNumber(summary.probi)